src/js/dataScores.js

```diff
@@ -0,0 +1,33 @@
+const dataScores = [
+  {
+    name: 'Name',
+    score: '100',
+  },
+  {
+    name: 'Name',
+    score: '20',
+  },
+  {
+    name: 'Name',
+    score: '50',
+  },
+  {
+    name: 'Name',
+    score: '78',
+  },
+  {
+    name: 'Name',
+    score: '125',
+  },
+  {
+    name: 'Name',
+    score: '77',
+  },
+  {
+    name: 'Name',
+    score: '42',
+  },
+];
+
+/* eslint-disable */
```

Comment:
- [ ] Same here, it is always a good practice to make sure you fix all linter errors instead of disabling them, A linter is great for identifying errors when you use standard rules. Remember, a linter analyzes your code for stylistic and programming errors against the rules it knows. 💯 😉

Reply:
Kindly, I suggest you follow the previous reviewer's advice and maybe disabling linters messages and follow the linter rules to solve your problem